ddtrace/debugging/_signal/snapshot.py

```diff
@@ -182,6 +182,8 @@ def exit(self, retval, exc_info, duration):
         _locals = []
         if exc_info[1] is None:
             _locals.append(("@return", retval))
+        else:
+            _locals.append(("@exception", exc_info[1]))
```

Comment:
why do add @exception to locals here? do we want to capture exc as "@exception" ?

Reply:
I've added this because we were already capturing the return value as `@return` in locals. I don't think it hurts to have it. Don't the other runtimes do this?

Reply:
From offline discussion: we decided to leave this change out for now, as there is no equivalent in other runtimes at the moment.